roofit/roofitcore/inc/RooAbsPdf.h

```diff
@@ -244,7 +244,7 @@ class RooAbsPdf : public RooAbsReal {
   }
   virtual Double_t expectedEvents(const RooArgSet* nset) const ; 
   /// Return expected number of events to be used in calculation of extended likelihood.
-  virtual Double_t expectedEvents(const RooArgSet& nset) const {
+  double expectedEvents(const RooArgSet& nset) const {
```

Comment:
Note that you might break user code here if people were deriving from this class.
It's unlikely that somebody overrode this, but a little comment might be helpful, even if it's just `\see <funcSignature>.`

Reply:
Note that the minimal info that doxygen needs to create an automatic link should be
`expectedEvents(const RooArgSet*) const`, i.e. all parameters with all `const` etc, but not the parameter names.